### ISSUE TITLE
[Project] Dialog to pick image from gallery or from camera

### ISSUE THREAD

**Issue by theohdv** [lucasferreira/react-native-webview-android#88]
Hello,

I'm using this component instead of the react native's one because it enables to upload file from the device storage.

I noticed that it does not offer the possibility to pick an image directly from camera. I think this feature could be useful and not very difficult to implement.

Do you know if someone is working on that now?

**Comment by guyzmo** [comment]
@theohdv have you found something that does that, or someone that implemented it?

**Comment by theohdv** [comment]
No, I'm still using this component without the feature. But it could be a great feature to add!

**Comment by guyzmo** [comment]
ok, I'm actually trying out https://github.com/dahjelle/react-native-android-webview-file-image-upload which *looks like* doing it. I'm implementing it directly in my project, instead of making it an external package… If it's working I might have a look at how to build it as a package.

**Comment by theohdv** [comment]
Okay it's a good idea, keep me informed 👍
